File: packages/sapplauz_core/lib/models/create_order_request.dart
class CreateOrderRequest {
  final int institutionId;
  final List<OrderItemRequest> orderItems;

  CreateOrderRequest({
    required this.institutionId,
    required this.orderItems,
  });

  Map<String, dynamic> toJson() {
    return {
      'institutionId': institutionId,
      'orderItems': orderItems.map((item) => item.toJson()).toList(),
    };
  }
}

class OrderItemRequest {
  final int performanceId;
  final int quantity;

  OrderItemRequest({
    required this.performanceId,
    required this.quantity,
  });

  Map<String, dynamic> toJson() {
    return {
      'performanceId': performanceId,
      'quantity': quantity,
    };
  }
}

File: backend/SApplauz.Shared/DTOs/CreateOrderRequest.cs
namespace SApplauz.Shared.DTOs;

public class CreateOrderRequest
{
    public int InstitutionId { get; set; }
    public List<OrderItemRequest> OrderItems { get; set; } = new();
}

public class OrderItemRequest
{
    public int PerformanceId { get; set; }
    public int Quantity { get; set; }
}






